R/auditCommonFunctions.R

```diff
@@ -815,7 +815,7 @@ gettextf <- function(fmt, ..., domain = NULL) {
       }
       if (options[["bayesian"]] && options[["expected_type"]] == "expected_abs" && options[["prior_method"]] %in% c("impartial", "arm")) {
         # Error if the prior construction method does not match the sampling objective
-        parentContainer$setError(gettext("The expected errors must specified relative to the total population to construct a prior distribution using this method."))
+        parentContainer$setError(gettext("In order to construct a prior distribution using risk assessments, specify the expected errors as a percentage of the total population."))
```

Comment:
Now the note says that it uses risk assessment, but I see that the message can be also triggered with `prior_method == "impartial"`.

Do you want to change the message to either 1) say "impartial" or "risk assessment" depending on the option selected, or 2) say something more generic? I don't mind either way (slightly prefer 1)), but I will leave that up to you.

Reply:
Great feedback!